src/components/LikeButtonV2/LikeButtonV2.vue

```diff
@@ -343,21 +358,37 @@ export default {
       }
     },
     superLikeIconStroke() {
+      if (this.state === 'unsuperlikeable') {
+        return '#9b9b9b';
+      }
+
       if (this.cooldown) {
         return this.hasSuperLiked ? '#50e3c2' : '#e6e6e6';
       }
       return '#28646e';
     },
     superLikeIconFill() {
-      if (this.state === 'superlikeable-cooldown') {
-        return '#50e3c200';
+      switch (this.state) {
+        case 'unsuperlikeable':
+        case 'superlikeable-cooldown':
+          return '#50e3c200';
+
+        default:
+          return '#50e3c2';
       }
-      return '#50e3c2';
     },
   },
   methods: {
     onClick() {
-      this.startClapBitsAnimation();
+      if (
+        ![
+          'unsuperlikeable',
+          'cooldown',
+          'superlikeable-cooldown',
+        ].some(s => s === this.state)
```

Comment:
`!['unsuperlikeable', 'cooldown', 'superlikeable-cooldown'].includes(this.state)` seems better